Single value wrapping (#6054)
confluentinc/ksql: docs/operate-and-deploy/installation/server-config/config-reference.md

[CREATE STREAM AS SELECT](../../../developer-guide/ksqldb-reference/create-stream-as-select.md)
statements.

If not set and no explicit value is provided in the statement, the value format's default wrapping 
is used.

> When set to the `true` ksqlDB serializes the column value nested within a JSON object, Avro record,

Reviewer: ```suggestion
When set to `true`, ksqlDB serializes the column value nested within a JSON object, Avro record,
```

[CREATE STREAM AS SELECT](../../../developer-guide/ksqldb-reference/create-stream-as-select.md)
statements.

 !!! note

>    Not all formats support wrapping and unwrapping. If you use a format that does not support

Reviewer: ```suggestion
   Not all formats support wrapping and unwrapping. If you use a format that doesn't support
```

>    the default value you set, the format will ignore the setting. For information on which formats

Reviewer 2: It doesn't look like the docs specify what the default value for each format is.

Also, where is the default added in the code right now? Is it hard-coded to be wrapped (if SerdeOptions is empty) somewhere in the planner?

Author: > It doesn't look like the docs specify what the default value for each format is.

Yep, that's on my todo list - next PR!

> Also, where is the default added in the code right now? Is it hard-coded to be wrapped (if SerdeOptions is empty) somewhere in the planner?

If the user doesn't supply the option then nothing is stored in the plan.  Currently, this defaults to wrapped, (code in GenericRowSerde and GenericKeySerde), but the next PR will change this, so that the format sets the default.

>    the default value you set, the format will ignore the setting. For information on which formats

Reviewer: ```suggestion
   the default value you set, the format ignores the setting. For information on which formats
```
